src/transformers/models/nllb/tokenization_nllb.py

```diff
@@ -225,12 +221,44 @@ def __setstate__(self, d):
 
     @property
     def vocab_size(self):
-        return len(self.sp_model) + len(self.lang_code_to_id) + self.fairseq_offset + 1  # Plus 1 for the mask token
+        return len(self.sp_model) + self.fairseq_offset
```

Comment:
That is expected, because the length (`len(tokenizer)`) will include the special tokens that are added. The vocab size should only include tokens part of the actual vocab (no extra tokens)